### PR TITLE
test: add stmt flow test cases

### PR DESCRIPTION
**Pull request by cfzjywxk**
Add `README.md` file and some basic `stmtflow` test cases.

### CODE REVIEW

**Comment by cfzjywxk** [comment]
@you06
PTAL

File: stmtflow/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# The stmtflow test tool
+
+
+## Introduction
+
+The `stmtflow` test [tool](https://github.com/zyguan/tidb-test-util) is used to run a suite of **sql tests** on the tinysql/tinykv cluster **concurrently**. 
+In the summer schoolproject, we could use this tool to write specific test cases with multiple connections or sessions to test the concurrency results of our
+transaction engine.
```
Comment (you06) [inline stmtflow/README.md]:
```suggestion
In the summer school project, we could use this tool to write specific test cases with multiple connections or sessions to test the concurrency results of our transaction engine.
```

File: stmtflow/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+The `stmtflow` test [tool](https://github.com/zyguan/tidb-test-util) is used to run a suite of **sql tests** on the tinysql/tinykv cluster **concurrently**. 
+In the summer schoolproject, we could use this tool to write specific test cases with multiple connections or sessions to test the concurrency results of our
+transaction engine.
+
+
+## How to use it
+
+```
+https://github.com/zyguan/tidb-test-util.git
```
Comment (you06) [inline stmtflow/README.md]:
```suggestion
git clone https://github.com/zyguan/tidb-test-util.git
```

File: stmtflow/test_in_txn_read.t.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/* init */ drop table if exists t1;
+/* init */ create table t1 (c1 int key, c2 int, c3 int, key k1(c3));
+/* init */ insert into t1 values (1, 1, 1);
+
+/* t1 */
```
Comment (you06) [inline stmtflow/test_in_txn_read.t.sql]:
```suggestion

```

File: stmtflow/test_snap_read.t.sql [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* init */ drop table if exists t1;
+/* init */ create table t1 (c1 int key, c2 int, c3 int, key k1(c3));
+/* init */ insert into t1 values (1, 1, 1);
+
+/* t1 */
```
Comment (you06) [inline stmtflow/test_snap_read.t.sql]:
```suggestion

```

**Review by you06 — APPROVED** [review]
LGTM

**Pull request merged**